packages/makeup-modal/src/index.js

```diff
@@ -3,12 +3,67 @@
 const keyboardTrap = require('makeup-keyboard-trap');
 const screenreaderTrap = require('makeup-screenreader-trap');
 
+let hoistEl;
+let hoistedElementPlaceholder;
+let containerDiv;
```

Comment:
Let's rename `containerDiv` to 'inertContentEl` so that its purpose is more clear (anything that is not inside of the modal is considered "inert").